### ISSUE TITLE
Xlwings vba edit error

### ISSUE THREAD

**Issue by awheelr** [xlwings/xlwings#2088]
OS:
Windows 11

Versions
0.28.5

Issue:

When using `xlwings vba edit` everytime I save the file and check the VBA Editor it adds `Attribute VB_Exposed = False` to the top of my code which causes a syntax error. Not sure why it's not ignoring this attribute like every other one. It might be a user error on my end not sure where else to ask though. Thanks!

**Comment by fzumstein** [comment]
The logic to ignore these lines is probably too simple right now. Would you be able to share an example workbook that exposes this issue?

**Comment by awheelr** [comment]
Okay, I have an example workbook that I've made that recreates the problem which is hopefully explained well below. I've also noticed while doing so that it seems to be something to do with the lines `ShowModal =  0` and `WhatsThisHelp = -1`.  I don't think it's anything specific between these two settings because if you go to use `xlwings vba import` it updates as it should. However when saving with CTRL-S using `xlwings vba edit` it seems it doesn't ignore those extra two lines so even if you added two comments for example instead of `ShowModal` and `WhatsThisHelp` it would still create this issue. I assume you are correct in the idea that the logic to ignore these lines is too simple. When creating a new workbook the defaults looked like this and it worked fine with no added line in the excel editor like before:

```vb
' This works perfectly
VERSION 5.00
Begin {C62A69F0-16DC-11CE-9E98-00AA00574A4F} UserForm1 
   Caption         =   "UserForm1"
   ClientHeight    =   3015
   ClientLeft      =   120
   ClientTop       =   465
   ClientWidth     =   4560
   OleObjectBlob   =   "UserForm1.frx":0000
   StartUpPosition =   1  'CenterOwner
End
Attribute VB_Name = "UserForm1"
Attribute VB_GlobalNameSpace = False
Attribute VB_Creatable = False
Attribute VB_PredeclaredId = True
Attribute VB_Exposed = False
```
However, when adding the two lines `ShowModal =  0` and/or `WhatsThisHelp = -1` it places the two last attributes in code editor.

```vb
' This causes an error
VERSION 5.00
Begin {C62A69F0-16DC-11CE-9E98-00AA00574A4F} UserForm1 
   Caption         =   "UserForm1"
   ClientHeight    =   9720.001
   ClientLeft      =   60
   ClientTop       =   210
   ClientWidth     =   14190
   OleObjectBlob   =   "UserForm1.frx":0000
   ShowModal       =   0   'False
   StartUpPosition =   2  'CenterScreen
   WhatsThisHelp   =   -1  'True
End
Attribute VB_Name = "UserForm1"
Attribute VB_GlobalNameSpace = False
Attribute VB_Creatable = False
Attribute VB_PredeclaredId = True
Attribute VB_Exposed = False
```

This is just my observation I don't know much about this project but I hope it helps in fixing this bug because I would love to use this feature on my current project in excel. :)

**Comment by fzumstein** [comment]
Right, currently it just cuts off a hardcoded number of lines (I don't really use Forms), so will have to improve this. Thanks for reporting!

**Comment by awheelr** [comment]
Awesome! This was the first bug I've ever submitted hopefully I didn't go overboard and good luck with solving the problem!

**Comment by HansThorsager** [comment]
Any news on this issue? Right now I'm facing it too and would really like to manage the VBA code in my forms through VSCode as well as I do with my modules

**Comment by Gallivant** [comment]
Felix, firstly thanks so much for the work you've done, as I too hate the VBA editor. Secondly, is it possible to upvote this issue, as I get the same problems, when using vba edit, with: 
1. Attribute mWkSht.VB_VarHelpID = -1, when a global is declared WithEvents; and 
2. Attribute Element.VB_UserMemId = 0, when a class property is declared as the default property
 
However, the issue doesn't arise with vba import.

Thanks again

**Comment by fzumstein** [comment]
I am happy to accept a pull request for this, but I personally don't have time for this or the motivation, as the xlwings source code doesn't have this issue.
